kv/local_test_cluster.go

```diff
@@ -146,5 +122,8 @@ func (ltc *LocalTestCluster) Start(t util.Tester) {
 
 // Stop stops the cluster.
 func (ltc *LocalTestCluster) Stop() {
+	if r := recover(); r != nil {
```

Comment:
Ok, I'm going to take them out. For debugging spurious errors (where we currently have to search the goroutine stacks) it seems like we should disencourage using `Fatal{,f}` (or make really clear how to debug timeouts during tests).